app/components/post_list/scroll_to_end_view.tsx

```diff
@@ -25,7 +24,7 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
         buttonStyle: {
             position: 'absolute',
             alignSelf: 'center',
-            bottom: -100,
```

Comment:
this was to give it enough space so it does not appear below the keyboard at any point when not shown, is that still the case with these changes ?